<<<<<<<< HEAD:recipes-kernel/linux/linux-skov/patches/1601-Release-6.14-customers-skov-imx6-20250815-1.patch
From: =?UTF-8?q?Ulrich=20=C3=96lmann?= <u.oelmann@pengutronix.de>
Date: Fri, 15 Aug 2025 09:04:58 +0200
Subject: [PATCH] Release 6.14/customers/skov/imx6/20250815-1
========
From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Thu, 11 Sep 2025 09:01:34 +0200
Subject: [PATCH] Release 6.14/customers/skov/imx6/20250911-1
>>>>>>>> 5ed974c (linux-skov: update to 6.14/customers/skov/imx6/20250911-1 (v6.14 -> v6.14)):recipes-kernel/linux/linux-skov/patches/2001-Release-6.14-customers-skov-imx6-20250911-1.patch

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
<<<<<<<< HEAD:recipes-kernel/linux/linux-skov/patches/1601-Release-6.14-customers-skov-imx6-20250815-1.patch
index 48d2ce96398b..a3def0d53fe2 100644
========
index 48d2ce96398b..3fe288f9a781 100644
>>>>>>>> 5ed974c (linux-skov: update to 6.14/customers/skov/imx6/20250911-1 (v6.14 -> v6.14)):recipes-kernel/linux/linux-skov/patches/2001-Release-6.14-customers-skov-imx6-20250911-1.patch
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 14
 SUBLEVEL = 11
-EXTRAVERSION =
<<<<<<<< HEAD:recipes-kernel/linux/linux-skov/patches/1601-Release-6.14-customers-skov-imx6-20250815-1.patch
+EXTRAVERSION =-20250815-1
========
+EXTRAVERSION =-20250911-1
>>>>>>>> 5ed974c (linux-skov: update to 6.14/customers/skov/imx6/20250911-1 (v6.14 -> v6.14)):recipes-kernel/linux/linux-skov/patches/2001-Release-6.14-customers-skov-imx6-20250911-1.patch
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
